rfcs/QueryLevelNullability.md

```diff
@@ -1,100 +1,158 @@
 # RFC: Query Level Nullability 
 
-**Proposed by:** 
+**Proposed by:**
+
+- [Alex Reilly](<social or github link here>) - Yelp iOS
```

Comment:
I put you at the top since you're the primary editor (and alphabetized everyone else)